### DISCUSSION TITLE
v0.6.0 alpha pre-release 2

### DISCUSSION THREAD

**Discussion by Correct-Syntax** [GimelStudio/GimelStudio#132] [Announcements]
Welcome to the **second** pre-release of the v0.6.x series of Gimel Studio!

This pre-release has some more of the basic functionality implemented and is mainly meant as a test version.

It is important to note that this is a pre-release and thus is by no means feature complete (thus this is not the MVP version). There are also some known issues, so just know that many things will change from what is currently here.

Let us know any feedback you have! Thanks.

(Only a Windows binary is provided for this release. Enjoy. 😃)

<hr /><em>This discussion was created from the release <a href='https://github.com/GimelStudio/GimelStudio/releases/tag/v0.6.0-alpha-2'>v0.6.0 alpha pre-release 2</a>.</em>

**Comment by llluuulll** [comment]
Hi,

Nice alpha, it is a promissing start.

Some quick observations :

The dpi awareness do not work properly (with my 4k screen setup) for the nodes and all the icons and images.

![image1](https://user-images.githubusercontent.com/56773214/154738721-3880e586-e664-4827-ab96-30ccaaa2485b.jpg)

All the keyboard shortcuts don't work for me, obligated to use the context menu to use the program.
The ui redraw get very laggy when using the ui splitters to slide the right and the bottom panels.

**Reply by Correct-Syntax** [reply]
Hi, thanks for the feedback. 😄  Yes, the hidpi scaling is definitely a problem. I had similar issues on another hd monitor https://github.com/GimelStudio/GimelStudio/issues/90 but we have yet to fix it.

> keyboard shortcuts don't work for me

Technically, I don't think any of the keyboard shortcuts are implemented as of yet (except Shift+A for the node graph)

> The ui redraw get very laggy when using the ui splitters to slide the right and the bottom panels

I think that is to be expected. Not sure how to improve that as the splitters are from wxPython.

**Comment by llluuulll** [comment]
Yes, the laggyness when sliding seem to be a sikness of gui python in general.
Tkinter suffer of the same problem.

**Comment by llluuulll** [comment]
Hi,

About the splitters, luckily in wxPython you can try this.
splitter = wx.SplitterWindow(self, style=wx.SP_LIVE_UPDATE)
This should fix the problem.

**Reply by Correct-Syntax** [reply]
Thank you
